gprofiler/main.py

```diff
@@ -230,8 +239,23 @@ def _strip_container_data(collapsed_data):
             lines.append(line[line.find(';') + 1 :])
         return '\n'.join(lines)
 
+    def send_metadata(self):
```

Comment:
This blocks. It runs after profiling has started. This logic should run in a different thread - don't slow down profiling / harm its accuracy by any way.

Reply:
Removed in #134